<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>261</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableIter">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>211</width>
      <height>321</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>*begin()</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>*end()</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>*cbegin()</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>*cend()</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>*rbegin()</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>*rend()</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>empty()</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>front()</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>back()</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>data()</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Значение</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Содержимое веткора</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>capacity:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineCapacity">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>max_size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineMaxSize">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineSize">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>177</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonClear">
       <property name="text">
        <string>clear()</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonAt">
       <property name="text">
        <string>at()</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="buttonPopBack">
       <property name="text">
        <string>pop_back()</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="buttonReserve">
       <property name="text">
        <string>reserve()</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="buttonPushBack">
       <property name="text">
        <string>push_back()</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="buttonResize">
       <property name="text">
        <string>resize()</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>88</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>743</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
